Automation Examples/070.md

```diff
@@ -53,13 +58,15 @@ foreach ♥mail in ♥mails
 end foreach
 ```
 
-#### 6. Send your .csv file to yourself
+### 7. Send your .csv file to yourself
+
+It is a good practice to store usernames and passwords in Credential Container. In order to add a credential, choose Tools/Credential Container..., click "Add" and in the "Key" field insert the name of your value, for example "mail:address". Remember to have a key containing the `:` sign. In the "Value" and "Retype value" fields, insert for example your mail address. To the same with a mail password.
 
-Make sure to provide your correct credentials.
+See below how to access those values using the `♥credential` special variable.
 
 ```G1ANT
 list.create text ♥path result ♥attachments
-mail.smtp host ‴imap.gmail.com‴ login ‴user@gmail.com‴ password ‴p@$$w0rD‴ from ‴robot@gmail.com‴ to ‴robotuser@g1ant.com‴ subject ‴Csv attachment‴ body ‴Hi, here's your mail info‴ attachments ♥attachments
+mail.smtp host ‴poczta.o2.pl‴ login ♥credential⟦mail:address⟧ password ♥credential⟦mail:password⟧ from ♥credential⟦mail:address⟧ to ‴robotuser@g1ant.com‴ subject ‴Csv attachment‴ body ‴Hi, here's your mail info‴ attachments ♥attachments
```

Comment:
I would suggest to use more anonimized smtp host instead of "poczta.o2.pl"

Reply:
maybe another credential variable instead of a visible host?

Reply:
Sure. Just make sure people will be instructed what to put there